bind_paths_and_mounts.rst

```diff
@@ -278,8 +276,8 @@ container.
 .. note::
```

Comment:
This `note` section can be removed completely since it is just talking about specifics of old singularity behavior between versions